docusaurus.config.js

```diff
@@ -37,14 +37,14 @@ const config = {
           // Please change this to your repo.
           // Remove this to remove the "edit this page" links.
           editUrl:
-            'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+            'https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/tree/main/docs/',
```

Comment:
Edit url generated invalid URLs, path end `docs/` not needed

```suggestion
            'https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/tree/main/',
```